src/styles/images/index.md.njk

```diff
@@ -25,10 +25,30 @@ If your image represents something physical, such as a letter, document or credi
 
 {{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
 
+
 ## Alt text
 
-All images must have an alt text. You should include a description of the image inside the tag to help users who are unable to see it.
+All images must have alt text. This is a description of an image that tells people what information the image provides. It is read out by screen readers or displayed if an image does not load or if images have been switched off.
```

Comment:
I would change `description of an image` to `meaning of an image`. The same image used in two different contexts can mean something different. For example, an icon of an eye usually means "Show" or "Preview" but a pure description would say "eye" which is less useful.

Reply:
I think that's better. I guess it's also supported by the below "describe the content and function of the image".
Although this is a point misunderstood by a lot of people, so might be worth making as clear as possible. I like how the GOV.UK guidance says somewhere (cannot find the link right now) to pretend to be on the phone and read the page aloud and how you would describe an image then.